ide/db/src/org/netbeans/api/db/explorer/ConnectionManager.java

```diff
@@ -187,6 +189,19 @@ public void removeConnection(DatabaseConnection dbconn) throws DatabaseException
 
         ConnectionList.getDefault().remove(dbconn.getDelegate());
     }
+    
+    /**
+     * Retrieves a list of database schemas available for given connection.
+     * 
+     * @param dbconn
+     * @return List of schemas or null if connection failed
```

Comment:
I usually use `{@code null}` to print it in a different font. Shouldn't an exception be raised if the connection fails?